features/support/env.rb

```diff
@@ -12,3 +15,42 @@
     @announce_env = true
   end
 end
+
+module AppendHelpers
+  def append_to(path, contents)
+    cd('.') do
+      File.open(path, "a") do |file|
+        file.puts
+        file.puts contents
+      end
+    end
+  end
+
+  def append_to_gemfile(contents)
+    append_to('Gemfile', contents)
+  end
+end
+
+module MinitestHelpers
+  def minitest_identifier
+    if rails_gte_4_1?
+      :minitest_5
+    else
+      :minitest_4
+    end
+  end
+end
+
+module RailsHelpers
+  def rails_version
+    Gem::Version.new(Bundler.definition.specs['rails'][0].version)
```

Comment:
I don't think a type of quote is being used consistently in this project, although single quotes seems to be used more often. I'm going to ignore these warnings.